src/components/TitleTransferPanel/TitleView/TitleView.tsx

```diff
@@ -13,12 +12,13 @@ interface TitleViewProps {
 
 export const TitleView = ({ role, address }: TitleViewProps) => {
   const addressHref = makeEtherscanAddressURL(address);
-  const issuerName = get(registry.issuers, `${address}`) ? get(registry.issuers, `${address}`).name : null;
+  const { getIdentifier } = useAddressBook();
+  const name = getIdentifier(address);
 
   return (
     <div className={css["title-view"]}>
-      <AddressInfo title={role} name={issuerName}>
-        <ExternalLink name={address} href={addressHref} />
+      <AddressInfo title={role} name={name}>
+        {address && <ExternalLink name={address} href={addressHref} />}
```

Comment:
why need to assert address? in the props it shows that it's always present

Reply:
mmm cos for bene and holder address they are being fetched asynchronously, i wanted it not to render anything at all so that the skeleton css :empty will work as expected